src/table/styling-utils.js

```diff
@@ -15,10 +15,14 @@ export function getHeadStyle(layout, theme) {
 
 export function getBodyStyle(layout, theme) {
   const content = layout.components?.[0]?.content;
+  const hoverColor = layout.components?.[0]?.hoverColor;
+  const hoverFontColor = layout.components?.[0]?.hoverFontColor;
   return content
     ? {
         fontColor: getColor(content.fontColor, '#404040', theme),
         fontSize: content.fontSize || '14px',
+        hoverBackGroundColor: `${getColor(hoverColor, 'blue', theme)} !important`,
+        hoverFontColor: getColor(hoverFontColor, 'black', theme),
```

Comment:
Need more appropriate defaults here. Some kind of light gray as background, and no change to the font color

Reply:
thanks fixed